<?xml version="1.0"?>
<launch>
    <node name="end_effector_servo" pkg="ee_servo2" type="ee_servo2_node"
        output="screen">

        <!-- 只解算到腕关节 -->
        <param name="chain_end" value="link4" />
        <param name="add_fake_joint_x" value="false" />
        <param name="add_fake_joint_y" value="true" />
        <param name="add_fake_joint_z" value="true" />
        <param name="add_fake_joint_num" value="2" />

        <!-- 解算到末端 -->
        <!-- <param name="chain_end" value="fake_link6" />
        <param name="add_fake_joint_x" value="false" />
        <param name="add_fake_joint_y" value="false" />
        <param name="add_fake_joint_z" value="false" />
        <param name="add_fake_joint_num" value="0" /> -->
    </node>
</launch>